config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
Whats this file about?

Reply:
We can specify the information needed to access our database here for different environments such as the `development`, `test` & `production`.

[Question]
1. What exactly is an adapter that we can specify in here?
2. Are we only able to use relational database here for Rails?